mypy/plugins/attrs.py

```diff
@@ -226,7 +226,7 @@ def _determine_eq_order(ctx: 'mypy.plugin.ClassDefContext') -> bool:
         order = eq
 
     if eq is False and order is True:
-        ctx.api.fail("eq must be True if order is True", ctx.reason)
+        ctx.api.fail('eq must be "True" if order is "True"', ctx.reason)
```

Comment:
No need for quotes here, I think just plain True reads more clearly as a Python literal